usautobuild/utils.py

```diff
@@ -0,0 +1,55 @@
+import io
+import selectors
+import subprocess
+
+from typing import Iterator
+
+__all__ = ("iterate_output",)
```

Comment:
what does this mean?

Reply:
`__all__` is list of objects to export with star import but i mainly use it to help linters find public objects in modules https://docs.python.org/3/tutorial/modules.html#importing-from-a-package